### ISSUE TITLE
Document how whitelisted repos are fed to the crawler

### ISSUE THREAD

**Issue by alranel** [italia/developers-italia-onboarding#34]
Do we generate a yml file? Do we expose APIs that the crawler consumes?

**Comment by libremente** [comment]
Flow: 
* <- insert registration info
* -> PEC
* <- confirm registration
* Store info in `private/data/whitelist.db.json` in json format, see:
```json
{
  "registrati": [
    {
      "referente": "pluto",
      "ipa": "c_a123",
      "url": "https://github.com/undefined",
      "pec": "protocollo.comunemaramao@pec.it"
    },
    {
      "referente": "pluto",
      "ipa": "c_a123",
      "url": "https://gitlab.com/undefined",
      "pec": "protocollo.comunemaramao@pec.it"
    }
  ]
}
```
As you may see, the same PA may insert different repos.
When asking for the `/repo-list` path, this file is served:
```yaml
---
  registrati: 
    - 
      ipa: "c_a123"
      url: "https://github.com/undefined"
      pec: "protocollo.comunemaramao@pec.it"
    - 
      ipa: "c_a123"
      url: "https://gitlab.com/undefined"
      pec: "protocollo.comunemaramao@pec.it"
```
@alranel 
I'm adding this stuff to a proper section in the README

**Comment by libremente** [comment]
Fixed in #37. Closing.
